packages/api/AlvTimeWebApi/Requests/TaskFavoriteRequest.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace AlvTimeWebApi.Requests
+{
+    public record TaskFavoriteRequest(int Id, bool Favorite, bool? Locked, string Name, decimal? CompensationRate);

```

Comment:
Hmmm, skal Locked og CompensationRate være med her? (Antar at dette går på om en task er favoritt eller ikke. Har ikke lest gjennom all koden, vil bare høre 🙂)

Reply:
Nei, strengt tatt trenger ikke Locked og CompensationRate å være med siden det ikke brukes til å oppdatere favoritt, men frontend sender hele tasken inn så for å slippe å gjøre endringer frontend så er det med her også